cli/README.md

```diff
@@ -31,15 +31,15 @@ The site build bundles `metadata.properties` along with the archetype catalog in
 This file is hosted on helidon.io. For example:
 https://helidon.io/cli-data/2.2.2/cli-data.zip
 
-In addition to cli-data, the site release also creates a `latest` file
-containing the latest Helidon version number:
-https://helidon.io/cli-data/latest
+In addition to cli-data, the site release also creates a `versions.xml` file
```

Comment:
Instead of changing this to document only the new mechanism we should document all of them.
Can you structure this README to document the old (current) metadata mechanism. Only make a note about the multiple trick we used in 3.x.